### DISCUSSION TITLE
[PSD8] Exercise 2.28/2.29

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#161] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210059987-a82ffc1a-53b2-405e-a617-f6e168394cbc.png)

**Comment by Elia-Belli** [comment]
![Kmap](https://user-images.githubusercontent.com/107944476/211626234-f1c97227-3972-4c45-bc49-a5d7bef2ba2b.png)
$Y =A \bar D + AB + AC$
![CC2](https://user-images.githubusercontent.com/107944476/212954769-30fcb768-76c0-4bb6-8766-d3dcd1e5b0c5.jpg)

**Comment by fraxb04** [comment]
corretto
